streams/javahippie.json

```diff
@@ -0,0 +1,26 @@
+{
+  "twitch_handle": "javahippie",
+  "youtube_handle": "javahippie",
+  "twitter_handle": "javahippie",
+  "streamer_timezone": "CEST",
+  "schedule": [
+    {
+      "title": "BreakPoint - Functional Friday Lunch Break",
+      "start_date_utc": "2020-08-28T10:30",
+      "duration_in_minutes": 30,
+      "platform": "twitch",
+      "stream_link": "http://twitch.tv/javahippie",
+      "recurring": {
+        "is_recurring": true,
+        "series_end": "2020-12-31T22:00"
+      }
```

Comment:
Daily? (Bi-)Weekly? Monthly? :)

Reply:
Good catch. Had "weekly" in mind, but we could add some enums for that. Or CRON syntax :)

Reply:
On second thought, I don't think "recurring" is a good idea. That opens the door to stale entries, leading to a confusing schedule and announcement tweets (assuming they operate from the same JSON). It would also be tough for us to identify such entries (I don't want to be in the business of comparing anybody's streaming habits to their data in this repo).

Reply:
Agreed, removed "recurring" entirely